tasks/main.yml

```diff
@@ -1,2 +1,18 @@
 ---
-# tasks file for skeleton
+- name: Create the /var/cyhy/admiral directory
+  ansible.builtin.file:
+    mode: 0755
+    path: /var/cyhy/admiral
+    state: directory
+
+- name: Install the Docker Compose configuration
+  ansible.builtin.copy:
+    dest: /var/cyhy/admiral
+    mode: 0644
+    src: docker-compose.yml
+
+- name: Install the Docker Compose development configuration
```

Comment:
If the `docker-compose-dev.yml` file is removed this task must also be removed.

Reply:
Removed in [df4546d](https://github.com/cisagov/ansible-role-admiral/pull/3/commits/df4546d74d5cff17ebea1a154f8b574f5a6c03f7).